x-pack/solutions/security/plugins/security_solution/public/assistant/get_comments/stream/stream_observable.ts

```diff
@@ -9,7 +9,8 @@ import { concatMap, delay, finalize, Observable, of, scan, timestamp } from 'rxj
 import type { Dispatch, SetStateAction } from 'react';
 import type { PromptObservableState } from './types';
 import { API_ERROR } from '../translations';
-const MIN_DELAY = 35;
+
+const MIN_DELAY = 10;
```

Comment:
seeing a lot of spacing changes here making this confusing... is the only real change here to `MIN_DELAY`?